package.json

```diff
@@ -119,6 +120,8 @@
     "@types/uuid": "^8.3.1",
     "eslint": "^8.0.0",
     "eslint-plugin-jest": "^26.0.0",
+    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-security": "1.4.0",
```

Comment:
Waiting for https://github.com/microsoft/eslint-plugin-sdl/pull/24 to be published so we can remove these two lines.